### PR TITLE
Feat: Create the email confirmation component

### LINKED ISSUE #30: Feat: Create the email confirmation component
No need for any functionality, just the structure, and the correct markup. It will be injected inside the layout component #27

image

### PR DESCRIPTION
**Pull request by Layla-Hammoud**
Fixes #30

### CODE REVIEW

**Comment by Layla-Hammoud** [comment]
email configuration done

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,8 @@
 import logo from './logo.svg';
 import './App.css';
-
+import Configuration from './components/emailconfiguratin';
```
Comment (wisammechano) [inline src/App.js:3]: Please revert App.js after testing

Comment (wisammechano) [inline src/App.js]: please revert App.js after testing.

File: src/components/emailconfiguratin.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import signinlogo from '../assets/signinlogo.svg';
+const Configuration = () => {
```
Comment (wisammechano) [inline src/components/emailconfiguratin.js]: Please rename to `Confirmation` and the file name too should be EmailConfirmation.js

**Review by wisammechano — CHANGES_REQUESTED** [review]

File: src/components/EmailConfirmation.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+      <div className="m-7 pt-10">
+        <div className="text-5xl">Thank you Layla!</div>
+      </div>
+      <div className="mx-10 px-10 pt-10 mt-10">
+        <p className="text-xl">Please check your email to verify the account</p>
+      </div>
+    </div>
+  );
+};
+export default Configuration;
```
Comment (wisammechano) [inline src/components/EmailConfirmation.js]: I fixed this for now, but please pay attention to export correctly

**Review by wisammechano — APPROVED** [review]

**Pull request merged**